project/SparkBuild.scala

```diff
@@ -596,19 +596,17 @@ object Hive {
 }
 
 object Assembly {
-  import sbtassembly.AssemblyUtils._
-  import sbtassembly.Plugin._
-  import AssemblyKeys._
+  import sbtassembly.AssemblyPlugin.autoImport._
 
   val hadoopVersion = taskKey[String]("The version of hadoop that spark is compiled against.")
 
-  lazy val settings = assemblySettings ++ Seq(
+  lazy val settings = Seq(
```

Comment:
I don't know enough about the SBT changes to evaluate these, but you seem like you know what you're doing. :) Just checking that this change was on purpose, to remove assemblySettings.